HW35_MyPet.py

```diff
@@ -0,0 +1,69 @@
+
+"""
+    Tamagotchi: can create pets that have favourite food and toy, feed them and play fetch
+
+"""
+class MyPet:
+
+    FAVOURITE_TOY = "ball"
+    FAVOURITE_FOOD = "cheese"
+    ENERGETIC = True
```

Comment:
For me these fields seems rather fields of instance than fields of class.
If we could have only one MyPet instance, we could make it singleton, but in this case anyway better use them as instance fields.
If we could create several instances of MyPet, then each pet should have its own level of energy, favourite toy and food. Because now if one pet will play, all other pets will be tired